### ISSUE TITLE
Field Attribute in line 13 has trailing data

### ISSUE THREAD

**Issue by nazar-pc** [sdroege/sdp-types#2]
Here is a sample that produces an error from the title when calling `Session::parse()`:
<details>

```
v=0
o=- 673858621984976475 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE 0 1 2
a=msid-semantic: WMS 2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF
m=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126
c=IN IP4 0.0.0.0
b=AS:1000
a=rtcp:9 IN IP4 0.0.0.0
a=ice-ufrag:6hn9
a=ice-pwd:8iglOGaMeJO6oPl65z5QtSJQ
a=ice-options:trickle
a=fingerprint:sha-256 3A:96:6D:57:B2:C2:C7:61:A0:46:3E:1C:97:39:D3:F7:0A:88:A0:B1:EC:03:FB:10:A5:5D:3A:37:AB:DD:02:AA
a=setup:actpass
a=mid:0
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=extmap:5 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id
a=extmap:6 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id
a=sendrecv
a=msid:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF 3a1d399d-a80a-4024-8ac8-12955babbbbe
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:103 ISAC/16000
a=rtpmap:104 ISAC/32000
a=rtpmap:9 G722/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:106 CN/32000
a=rtpmap:105 CN/16000
a=rtpmap:13 CN/8000
a=rtpmap:110 telephone-event/48000
a=rtpmap:112 telephone-event/32000
a=rtpmap:113 telephone-event/16000
a=rtpmap:126 telephone-event/8000
a=ssrc:435068003 cname:lVFNW7wnHJf2lDhW
a=ssrc:435068003 msid:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF 3a1d399d-a80a-4024-8ac8-12955babbbbe
a=ssrc:435068003 mslabel:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF
a=ssrc:435068003 label:3a1d399d-a80a-4024-8ac8-12955babbbbe
m=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 100 101 102 125 127
c=IN IP4 0.0.0.0
b=AS:100000
a=rtcp:9 IN IP4 0.0.0.0
a=ice-ufrag:6hn9
a=ice-pwd:8iglOGaMeJO6oPl65z5QtSJQ
a=ice-options:trickle
a=fingerprint:sha-256 3A:96:6D:57:B2:C2:C7:61:A0:46:3E:1C:97:39:D3:F7:0A:88:A0:B1:EC:03:FB:10:A5:5D:3A:37:AB:DD:02:AA
a=setup:actpass
a=mid:1
a=extmap:14 urn:ietf:params:rtp-hdrext:toffset
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:13 urn:3gpp:video-orientation
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:12 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
a=extmap:11 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type
a=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing
a=extmap:8 http://tools.ietf.org/html/draft-ietf-avtext-framemarking-07
a=extmap:9 http://www.webrtc.org/experiments/rtp-hdrext/color-space
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=extmap:5 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id
a=extmap:6 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id
a=sendrecv
a=msid:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF 07a3fbf7-f486-4bcb-9284-62d7a742c5e8
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 goog-remb
a=rtcp-fb:96 transport-cc
a=rtcp-fb:96 ccm fir
a=rtcp-fb:96 nack
a=rtcp-fb:96 nack pli
a=rtpmap:97 rtx/90000
a=fmtp:97 apt=96
a=rtpmap:98 VP9/90000
a=rtcp-fb:98 goog-remb
a=rtcp-fb:98 transport-cc
a=rtcp-fb:98 ccm fir
a=rtcp-fb:98 nack
a=rtcp-fb:98 nack pli
a=fmtp:98 profile-id=0
a=rtpmap:99 rtx/90000
a=fmtp:99 apt=98
a=rtpmap:100 VP9/90000
a=rtcp-fb:100 goog-remb
a=rtcp-fb:100 transport-cc
a=rtcp-fb:100 ccm fir
a=rtcp-fb:100 nack
a=rtcp-fb:100 nack pli
a=fmtp:100 profile-id=2
a=rtpmap:101 rtx/90000
a=fmtp:101 apt=100
a=rtpmap:102 red/90000
a=rtpmap:125 rtx/90000
a=fmtp:125 apt=102
a=rtpmap:127 ulpfec/90000
a=ssrc-group:FID 807864019 2899882658
a=ssrc:807864019 cname:lVFNW7wnHJf2lDhW
a=ssrc:807864019 msid:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF 07a3fbf7-f486-4bcb-9284-62d7a742c5e8
a=ssrc:807864019 mslabel:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF
a=ssrc:807864019 label:07a3fbf7-f486-4bcb-9284-62d7a742c5e8
a=ssrc:2899882658 cname:lVFNW7wnHJf2lDhW
a=ssrc:2899882658 msid:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF 07a3fbf7-f486-4bcb-9284-62d7a742c5e8
a=ssrc:2899882658 mslabel:2XGYJZsCnFdqofToLt6Jr2STw9LDq06BSpgF
a=ssrc:2899882658 label:07a3fbf7-f486-4bcb-9284-62d7a742c5e8
m=application 9 UDP/DTLS/SCTP webrtc-datachannel
c=IN IP4 0.0.0.0
a=ice-ufrag:6hn9
a=ice-pwd:8iglOGaMeJO6oPl65z5QtSJQ
a=ice-options:trickle
a=fingerprint:sha-256 3A:96:6D:57:B2:C2:C7:61:A0:46:3E:1C:97:39:D3:F7:0A:88:A0:B1:EC:03:FB:10:A5:5D:3A:37:AB:DD:02:AA
a=setup:actpass
a=mid:2
a=sctp-port:5000
a=max-message-size:262144
```

</details>

**Comment by sdroege** [comment]
Thanks for reporting, I can confirm that and have a fix. Will write some more tests and create a PR later.

**Comment by sdroege** [comment]
See https://github.com/sdroege/sdp-types/pull/3

**Comment by sdroege** [comment]
Released 0.1.1 to crates.io with this fix. Thanks again for reporting!
